packages/connection/tests/php/test_Error_Handler.php

```diff
@@ -62,11 +62,11 @@ public function test_store_error() {
 
 		$stored_errors = $this->error_handler->get_stored_errors();
 
-		$this->assertEquals( 1, count( $stored_errors ) );
+		$this->assertSame( 1, count( $stored_errors ) );
```

Comment:
Random thought: Maybe we should have a sniff to prefer `assertCount` over things like this.